packages/slice-machine/src/modules/slices/index.ts

```diff
@@ -164,10 +169,15 @@ export function* renameSliceSaga({
       payload.newSliceName,
       payload.libName
     );
+    const { data: serverState } = (yield call(getState)) as SagaReturnType<
```

Comment:
I tried to change it manually in the reducer. The issue is that I had to change it lots of places because the name change will lead to a different path, therefore on top of `model.name` we also change `screenshotUrls`, `screenshotPaths` etc. Was worried of forgetting/missing some so decided it was best to refetch. 

Should i try to change it manually in the reducer?

Reply:
Spike inc so we'll say it's ok for now but we'll need to change it later.
Maybe the server could send it directly after the call to rename, as the env is already computed this could save us some time maybe.

Reply:
added a TODO to say that